docs/streams/developer-guide/app-reset-tool.html

```diff
@@ -0,0 +1,236 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<script><!--#include virtual="../js/templateData.js" --></script>
+
+<script id="content-template" type="text/x-handlebars-template">
+  <h1>Developer Guide for Kafka Streams API</h1>
+  <div class="sub-nav-sticky">
+    <div class="sticky-top">
+      <div style="height:35px">
+        <a href="/{{version}}/documentation/streams/">Introduction</a>
+        <a class="active-menu-item" href="/{{version}}/documentation/streams/developer-guide">Developer Guide</a>
```

Comment:
Since we are adding lots of more content into the developer guide section, it will make the already-very-large page even larger. Could we consider making a sub-nav under Developer guide? Otherwise if people wants to browse through the developer guide they'd need to use `next` / `prev` button which is very tedious.

We do not need to do this in this PR to keep the PR from growing indefinitely, but we may consider doing this right away after this PR?

Reply:
this PR will make subpages - the intention is NOT to make another long dev guide page.

Reply:
What I meant is actually for navigation: today within the developer guide users can only "move one section at a time", with a secondar nav bar they can then jump from one sub-section to another non-consecutive sub-section.

Reply:
@guozhangwang the landing page looks like this:
![image](https://user-images.githubusercontent.com/11722533/33910022-40ec8560-df42-11e7-9bb7-5c127b818c07.png)

Each of the child pages have a breadcrumb in the header which links back to the top-level Dev Guide/index page (`DOCUMENTATION
KAFKA STREAMS API
DEVELOPER GUIDE`):
![image](https://user-images.githubusercontent.com/11722533/33910108-7d032efa-df42-11e7-9e72-5f8fad09f146.png)